src/components/Capture/capture.js

```diff
@@ -67,17 +69,27 @@ class Capture extends Component {
   uploadCaptureToOnfido = () => {
     this.setState({uploadInProgress: true})
     const {validCaptures, method, side, token} = this.props
-    const capture = validCaptures[0]
+    const {blob, documentType, id} = validCaptures[0]
+    this.setState({captureId: id})
 
-    postToOnfido(capture, method, token, this.state.advancedValidation,
-      (apiResponse) => this.confirmAndProceed(apiResponse, capture.id),
-      this.onApiError
-    )
+    if (method === 'document') {
+      const data = { file: blob, type: documentType, side}
+      if (this.state.documentValidations) {
+        const validations = {'detect_document': 'error', 'detect_glare': 'warn'}
+        data['sdk_validations'] = JSON.stringify(validations)
```

Comment:
Minor: Serializing is what could be handled by the `onfidoApi` module, since it's logic that doesn't need to belong on the UI component.